Fix unreliable selector detection

The Python function `is_selector` is unreliable (probably due to module name and import changes).

In particular, this test is failing:
```sh
 _______________________ test_run_python_snippets[path32] _______________________
[gw0] linux -- Python 3.13.7 /home/runner/work/fancy-polars/fancy-polars/.venv/bin/python3
/home/runner/work/fancy-polars/fancy-polars/py-polars/tests/docs/test_user_guide.py:41: in test_run_python_snippets
    runpy.run_path(str(path))
<frozen runpy>:287: in run_path
    ???
<frozen runpy>:98: in _run_module_code
    ???
<frozen runpy>:88: in _run_code
    ???
/home/runner/work/fancy-polars/fancy-polars/docs/source/src/python/user-guide/expressions/column-selections.py:93: in <module>
    print(expand_selector(df, out))
/home/runner/work/fancy-polars/fancy-polars/.venv/lib/python3.13/site-packages/fancy_polars/selectors.py:194: in expand_selector
    raise TypeError(msg)
E   TypeError: expected a selector; found cs.temporal() instead.
```

--- a/py-polars/fancy_polars/selectors.py
+++ b/py-polars/fancy_polars/selectors.py
@@ -112,7 +112,7 @@ def is_selector(obj: Any) -> bool:
     True
     """
     # note: don't want to expose the "_selector_proxy_" object
-    return isinstance(obj, _selector_proxy_) and hasattr(obj, "_attrs")
+    return hasattr(obj, "_pyexpr") and hasattr(obj, "_attrs")
 
 
 # TODO: Don't use this as it collects a schema (can be very expensive for LazyFrame).

